includes/definitions/discovery/eatonups.yaml

```
@@ -22,6 +22,14 @@ modules:
                     num_oid: '.1.3.6.1.4.1.534.1.4.1.{{ $index }}'
                     descr: 'Output Load'
                     index: 'xupsOutputLoad.{{ $index }}'
+        power:
+            data:
+                -
+                    oid: xupsOutputWatts
+                    value: xupsOutputWatts
```

i might be wrong here but I think it should be
```suggestion
                    oid: xupsOutputTable
                    value: xupsOutputWatts
```